model/client4.go

```diff
@@ -2168,17 +2168,6 @@ func (c *Client4) SearchPostsWithParams(teamId string, params *SearchParameter)
 	}
 }
 
-// SearchPosts returns any posts with matching terms string including deleted channels.
-func (c *Client4) SearchPostsIncludeDeletedChannels(teamId string, terms string, isOrSearch bool) (*PostList, *Response) {
```

Comment:
I don't think we can remove this since we want backwards compatibility here

Reply:
It's for an experimental feature. So we can remove it.